site/pages/__snapshots__/dashboard.test.tsx.snap

```diff
@@ -1236,7 +1236,7 @@ exports[`pages > dashboard > should render correctly when there are only live di
                 >
                   <a
                     className="govuk-link"
-                    href="/review-disruption/44"
+                    href="/disruption-detail/44?return=%2Fdashboard"
```

Comment:
why %2F ?

Reply:
%2F is URI encoded value for forward slash `/`